modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ImportTranslationResultsMVCRenderCommand.java

```diff
@@ -56,12 +63,26 @@ public String render(
 			httpSession.getAttribute(
 				ImportTranslationResultsDisplayContext.class.getName()));
 
+		renderRequest.setAttribute(
+			FFBulkTranslationConfiguration.class.getName(),
+			_ffBulkTranslationConfiguration);
+
```

Comment:
In general, try to inject as few request variables as possible. It is better to inject the configuration into the display context (so you need to add the dependency where it is created, in `ImportTranslationMVCActionCommand`), and expose a new method in the display context named `isDownloadCSVReportEnabled()` or something like that.

Reply:
I'll do that